tests/resources/VCH-Util.robot

```diff
@@ -322,7 +322,8 @@ Run Secret VIC Machine Inspect Command
 
 Run VIC Machine Delete Command
     ${rc}  ${output}=  Run Secret VIC Machine Delete Command  %{VCH-NAME}
-    Wait Until Keyword Succeeds  6x  5s  Check Delete Success  %{VCH-NAME}
+    ${status}=  Run Keyword And Return Status  Wait Until Keyword Succeeds  6x  5s  Check Delete Success  %{VCH-NAME}
+    Run Keyword Unless  ${status}  Conditional Fatal Error  vic-machine delete has failed to remove the VCH: %{VCH-NAME} as expected
```

Comment:
This might be a request for a different Keyword, but I would more naturally phrase this with the same imperatives used in unit tests:
* assert
* require

Assert is a failure, require is a fatal failure. In this case we're changing the behaviour of assert to control whether it become fatal, so it wraps both the check of status and the appropriate handling on failure into a single keyword:
```
${status}=  Run Keyword And Return Status  Wait Until Keyword Succeeds  6x  5s  Check Delete Success  %{VCH-NAME}
Assert  ${status}  "vic-machine delete failed to remove %{VCH-NAME}"
```